apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/resources/Sgv2RowsResourceApi.java

```diff
@@ -30,6 +30,7 @@
  */
 @ApplicationScoped
 @Path("/v2/keyspaces/{keyspaceName}/{tableName}")
+@Encoded
```

Comment:
* What about the other resource classes?
* And what happens now if I send encoded keyspace and table name, I guess this fails on the bridge call as expected

Reply:
I suspect this might be the right annotation to use, but placement may be wrong: I think it would belong only next to:

```
         @PathParam("primaryKey")
          List<PathSegment> path,
```

and/or anything needed wrt annotations should be there. We do not want to prevent decoding of all parameter (query, path) values.

Reply:
FWTW, semantics for `@Encoded` are explained f.ex here:

https://docs.jboss.org/resteasy/docs/1.0.0.GA/userguide/html/_Encoded_and_encoding.html

Reply:
I tried applying the annotation to just the path on my first commit on this branch but it did not pass tests.